ui/installer/VCSA/install.sh

```diff
@@ -36,14 +39,10 @@ while IFS='' read -r line; do
     eval $line
 done < ./configs
 
-# check for the VC IP
-if [[ $VCENTER_IP == "" ]] ; then
-    echo "Error! vCenter IP cannot be empty. Please provide a valid IP in the configs file"
-    cleanup
-    exit 1
-fi
+# replace space delimiters with colon delimiters 
```

Comment:
Yeah I am aware of that, but in case some user manually updates it which should never be the case now that we have a warning message in the configs file, this still could work as a safety net.